core/pkg/artifacts/linker.go

```diff
@@ -24,8 +26,17 @@ func (al *ArtifactLinker) Link() error {
 	portfolioName := al.LinkArtifact.PortfolioName
 	portfolioEntity := al.LinkArtifact.PortfolioEntity
 	portfolioProject := al.LinkArtifact.PortfolioProject
+	organization := al.LinkArtifact.PortfolioOrganization
 	var portfolioAliases []gql.ArtifactAliasInput
 
+	var err error
+	if IsArtifactRegistryProject(portfolioProject) {
+		portfolioEntity, err = al.resolveOrgEntityName(portfolioEntity, organization)
```

Comment:
Will this method be called if we use the org-entity in the portfolio path too? In that case, won't it return the incorrect error of "cant find artifact under org"? Or did I misread the code 😅

Reply:
oh nevermind. your gql query should be able to handle that correctly. ignore this.